src/components/EventItem.js

```diff
@@ -8,16 +8,13 @@ const EventItem = ({ title, image, startDate, endDate, organisator, city, catego
   const end = moment(endDate).format("D MMM");
 
   return (
-    <Link to={`edit/${id}`}>
+    <Link to={`read/${id}`}>
       {image ? (
-        <div>
-          <img src={image} alt="Event" />
-        </div>
+        <img src={image} alt="Event" />
```

Comment:
Do you have access here to some more meaningful name that `Event`? Maybe name of that event or path? Let's consider an example where you have multiple events and all of those are displaying just `Event` as the alt parameter :<